ajax/libs/webvr-polyfill/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "webvr-polyfill",
+  "version": "0.9.39",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/googlevr/webvr-polyfill"
+  },
+  "authors": [
+    "Boris Smus <boris@smus.com>",
+    "Brandon Jones <tojiro@gmail.com>",
+    "Jordan Santell <jordan@jsantell.com>"
+  ],
+  "filename": "webvr-polyfill.min.js",
+  "description": "Use WebVR today, on mobile or desktop, without requiring a special browser build.",
+  "keywords": [
+    "vr",
+    "webvr"
+  ],
+  "license": "Apache-2.0",
+  "npmName": "webvr-polyfill",
+  "npmFileMap": [
+    {
```

Comment:
Let's use git auto-update config
Because there are 45 releases versions from npm, and 50 releases versions from GitHub.
As usual, we prefer to use more releases versions as the auto-update option.
Also, the newest version (v1.0.0) is from GitHub.

Let's modify from A to B.
- A
```js
  "npmName": "webvr-polyfill",
  "npmFileMap": [
    {
      "basePath": "build",
      "files": [
        "**/*"
      ]
    }
  ]
```
-B
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/googlevr/webvr-polyfill.git",
    "fileMap": [
      {
        "basePath": "build",
        "files": [
          "**/!(*license)*"
        ]
      }
    ]
  }
```
Thank you.